src/crimson/osd/pg.h

```diff
@@ -589,6 +589,7 @@ class PG : public boost::intrusive_ref_counter<
   interruptible_future<> repair_object(
     const hobject_t& oid,
     eversion_t& v);
+  void check_blocklisted_obc_watchers(ObjectContextRef &obc);
```

Comment:
`ObjectContextRef` is already a reference, no need to pass it by reference.

Reply:
Um, I think that `ObjectContextRef` is actually an intrusive ptr, I'm passing it by reference just to avoid creating a new ptr.